src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,112 @@
+import { Person } from '../types/Person';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PersonLink } from '../components/PersonLink';
+
+const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [loadingError, setLoadingError] = useState(false);
+
+  const preparePeopleData = (persons: Person[]) => {
+    return people.map(person => ({
+      ...person,
+      mother: persons.find(({ name }) => name === person.motherName),
+      father: persons.find(({ name }) => name === person.fatherName),
+    }));
+  };
+
+  const preparedPeople = preparePeopleData(people);
+
+  const { personSlug } = useParams<{ personSlug: string }>();
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then(fetchedPeople => {
+        setPeople(fetchedPeople);
+      })
+      .catch(() => {
```

Comment:
In the catch block, you should also catch a possible error, because now the error does not look informative, for example, you can do this:

```js
catch(error) { setIsError(`Something went wrong: ${error.message}`)}
```